tests/backup_test.py

```diff
@@ -32,9 +32,9 @@ def test_backup(s3_stub, mocker, datadir):
 
     patched_list_kind_deployment = mocker.patch("kubernetes.client.apis.apps_v1_api.AppsV1Api.list_namespaced_deployment", autospec=True)
     patched_list_kind_deployment.return_value = create_response_data(datadir.join('deploymentlist.json').strpath, 'V1DeploymentList')
-
+    
     patched = mocker.patch("kubernetes.client.apis.core_v1_api.CoreV1Api.read_namespaced_config_map", autospec=True)
```

Comment:
We may need to think about this if we want to test a config map that isn't the "cluster data"

Reply:
Yes, have run make tests, are they failing for you?

Reply:
it test config-map coredns too?